include/tsid/contacts/contact-two-frame-positions.hpp

```diff
@@ -68,7 +68,7 @@ class ContactTwoFramePositions : public ContactBase {
   virtual const ConstraintEquality& computeForceRegularizationTask(
       const double t, ConstRefVector q, ConstRefVector v, const Data& data);
 
-  const TaskSE3Equality& getMotionTask() const;
+  const TaskTwoFramesEquality& getMotionTask() const;
```

Comment:
Shouldn't the `getMotionTask` method return a `TaskMotion`?

Reply:
The pure virtual method `TaskMotion& getMotionTask` of the parent class `ContactBase` should be implemented in this class, shouldn't it?

Reply:
You're right. It works. I've recreated the same situation with this minimalistic code, and indeed the compiler doesn't complain. I didn't know you could do this. At this point I am fine with this PR and I am gonna merge it. Thanks again for the great contribution!

```
#include <iostream>
using namespace std;

class TaskMotion {};
class TaskTwoFramesEquality : public TaskMotion {};

class ContactBase {
    public:
    virtual const TaskMotion& getMotionTask() const = 0;
};

class ContactTwoFramePositions : public ContactBase {
    public:
    TaskTwoFramesEquality m_task;

    const TaskTwoFramesEquality& getMotionTask() const{
        cout<<"TaskTwoFramesEquality& getMotionTask"<<endl;
        return m_task;
    }
};

int main(){
    ContactTwoFramePositions c;
    const TaskMotion& task = c.getMotionTask();
    const TaskTwoFramesEquality& task2 = c.getMotionTask();
    return 0;
}
```